azure-pipelines.yml

```diff
@@ -41,50 +41,13 @@ variables:
     value: "ukhydrographicoffice/terraform-azure-powershell-unzip:1.3.7"
   - name: DeploymentPool
     value: "UKHO Ubuntu 1804"
-  - name: coverityPool
-    value: NautilusBuild
   - group: Covscan-vars
```

Comment:
This isn't needed now either. There's a separate PBI to remove the libraries from all the projects in Azure DevOps at a later date.

Reply:
Removed